packages/cubejs-client-react/index.d.ts

```diff
@@ -44,6 +44,31 @@ declare module '@cubejs-client/react' {
    */
   export const CubeProvider: React.FC<CubeProviderVariables>;
 
+  /**
+   * Cube.js context consumer
+   * ```js
+   * import React from 'react';
+   * import { CubeContext } from '@cubejs-client/react';
+   *
+   *
+   * export default function DisplayComponent() {
+   * const { cubejsApi } = React.useContext(CubeContext);
+   * const query = {
+   *   ...
+   * };
+   * const resultSet = await cubejsApi.load(query);
```

Comment:
You can't really use `await` directly in components. You should use `useEffect`

Reply:
ah good catch, had over simplified
added useEffect